lib/cisco_node_utils/cmd_ref/itd_device_group.yaml

```diff
@@ -0,0 +1,83 @@
+# itd_device_group
+---
+_exclude: [N3k, N5k, N6k, ios_xr]
+
+_template:
+  get_command: "show running-config all | section itd"
+  get_context:
+      - '/^itd device-group <name>/'
+      - '(?)/^node <ntype> <nname>/'
```

Comment:
Actually, itd_device_group_node has 2 extra properties (weight and hot_standby), the other properties of itd_device_group are all applicable to itd_device_group_node (only the context is different, we have an extra node context for the common properties) and I reused all the properties and methods in the ruby file by deriving itd_device_group_node from itd_device_group. This way I have kept the code to minimum. If we separate them into 2 files, then I have to keep almost the entire file same except add the other 2 properties in the new file. If you think, it is ok to repeat, I can break them into 2 files. We can also discuss over phone if you want. Let me know.